eslint.config.mjs

```diff
@@ -752,23 +752,30 @@ export default typescript.config([
     files: ['**/*.spec.{ts,js,tsx,jsx}', 'tests/js/**/*.{ts,js,tsx,jsx}'],
     plugins: jest.configs['flat/recommended'].plugins,
     rules: {
+      ...jest.configs['flat/recommended'].rules,
+      ...jest.configs['flat/style'].rules,
+
+      // `recommended` set this to warn, we've upgraded to error
       'jest/no-disabled-tests': 'error',
 
+      // `recommended` set this to warn, we've downgraded to off
       // Disabled as we have many tests which render as simple validations
       'jest/expect-expect': 'off',
 
       // Disabled as we have some comment out tests that cannot be
       // uncommented due to typescript errors.
-      'jest/no-commented-out-tests': 'off',
+      'jest/no-commented-out-tests': 'off', // TODO(ryan953): Fix violations then delete this line
 
       // Disabled as we do sometimes have conditional expects
-      'jest/no-conditional-expect': 'off',
-
-      // Useful for exporting some test utilities
-      'jest/no-export': 'off',
+      'jest/no-conditional-expect': 'off', // TODO(ryan953): Fix violations then delete this line
```

Comment:
43 instances of this, of the ones i spot-checked i think we can rewrite the tests to be more direct.